precompiles/parachain-staking/StakingInterface.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity >=0.8.3;
 
 /// @dev The ParachainStaking contract's address.
-address constant PARACHAIN_STAKING_ADDRESS = 0x0000000000000000000000000000000000000800;
+address constant PARACHAIN_STAKING_ADDRESS = 0x0000000000000000000000000000000000000407;
```

Comment:
is it possible to leave the same address than moonbeam ? It's just an address and we reduce the delta